### ISSUE TITLE
Docset path not correctly constructed

### ISSUE THREAD

**Issue by canatella** [dash-docs-el/dash-docs#13]
It seems for some docset, like `Python_3`, the install path is `<docsets-path>/Python 3` where the underscore has been replaced by a space, is this the case for all docsets ? I look at the json and XML but there's does not seem to be any mapping between the XML file name we use for the docset name and the extracted path. Maybe we should replace the `_` from the docset name ?

**Comment by kidd** [comment]
You're right, there is no pattern associated with the name of the docset and the file.  

As we wanted to be compatible with the original Dash folder, we can't do a lot of magic. Maybe try a "fuzzy find" on the elisp side?

**Comment by canatella** [comment]
Created a PR for a tentative fix.
